index.bs

```diff
@@ -1975,6 +1974,37 @@ offers guidance that should be considered in the development of new features, no
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3 id="new-data-formats">New Data Formats</h3>
+
+Always define a corresponding MIME type and extend existing APIs to support this type
+for any new data format.
+
+There are cases when a new capability on the web involves adding a new data format.
+This can be an image, video, audio, or any other type of data that a browser is expected
+to ingest. New formats should have a standardized MIME type, which is strictly validated.
+
+While legacy media formats do not always have strict enforcement for MIME types (and
+sometimes rely on peeking at headers, to workaround this), this is mostly for legacy
+compatibility reasons and should not be expected or implemented for new formats.
+
+It is expected that spec authors also integrate the new format to existing APIs, so that
+they are whitelisted in both ingress (e.g. decoding from a ReadableStream) and egress
+(e.g. encoding to a WriteableStream) points from a browser's perspective.
+
+For example. if you are to add an image format to the web platform, first add a new MIME
+type for the format. After this, you would naturally add a decoder (and presumably an
+encoder) for said image format to support decoding in HTMLImageElements. On top of this,
+you are also expected to add support to egress points such as HTMLCanvasElement.toBlob()
+and HTMLCanvasElement.toDataURL().
+
+For legacy reasons browsers support MIME type sniffing, but we do not recommend extending
+the [pattern matching algorithm](https://mimesniff.spec.whatwg.org/#image-type-pattern-matching-algorithm),
+and instead recommend enforcing strict MIME types for newer formats.
+
+New MIME types should be submitted to the Internet Engineering Steering Group (IESG) for review,
+approval, and registration with the Internet Assigned Numbers Authority (IANA).
```

Comment:
I think they only need to be registered.

Reply:
so just "should be registered with IANA"?